doc/stdlib/arrays.rst

```diff
@@ -187,7 +187,7 @@ Constructors
 
    Create an array of all zeros with the same element type and shape as ``A``\ .
 
-.. function:: ones([T::Type=Float64,] dims)
+.. function:: ones(type, dims)
```

Comment:
I restored this back to the way it was originally. Getting this to work with the `@eval` loop was basically impossible because `sysimg.jl` loads `string.jl` after `array.jl`.